xml/System.Threading/Thread.xml

```diff
@@ -530,7 +530,7 @@ Main thread: ThreadProc.Join has returned.  Press Enter to end program.
 > [!NOTE]
 > On versions of Microsoft Windows prior to Windows XP and Windows Server 2003, `maxStackSize` is ignored, and the stack size specified in the executable header is used.  
   
- If you specify a very small stack size, you might need to disable stack-overflow probing. When the stack is severely constrained, the probing can itself cause a stack overflow. To disable stack overflow probing, add the following to your application configuration file.  
+ If you specify a very small stack size, you might need to disable stack-overflow probing. When the stack is severely constrained, the probing can itself cause a stack overflow. To disable stack overflow probing, add the following to your application configuration file in a .NET Framework app.  
```

Comment:
I don't think this config section applies to .NET Core. My knowledge about this is limited, but AFAIK this is not configurable in .NET Core in `*.runtimeconfig.json`. Perhaps the previous statement is more accurate.

Reply:
@kouvel I've added "in a .NET Framework app" - so doesn't that make the addition more accurate since it excludes .NET Core?

Reply:
Ah yes I think I confused this with something else, this looks more accurate to me, thanks!